.github/workflows/static.yml

```diff
@@ -27,17 +27,33 @@ jobs:
     environment:
       name: github-pages
       url: ${{ steps.deployment.outputs.page_url }}
+
     runs-on: ubuntu-latest
+
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run webpack build
+        run: npx webpack --config webpack.prod.js
+
```

Comment:
This has not been tested. At least according to the readme for `v3` of the setup-node they mention version 18 as a major version they support so we should be good there. I tested on node version 19 but dont think I am doing anything that broke between those major versions.